js/README.md

```diff
@@ -0,0 +1,73 @@
+# Solido JS SDK
+
+Typescript SDK to facilitate interacting with the Solido program in JS/TS applications
+
+## Installation
+
+### Yarn
+
+```
+$ yarn add @chorusone/solido.js
+```
+
+### npm
+
+```
+$ npm install @chorusone/solido.js
+```
+
+## Development
+
+### Install packages
+```
+$ npm install
+or
+$ yarn
+```
+
+### Run TS watch script
+```
+$ npm run watch:ts
+```
+
+### Run tests watch script
+```
+$ npm run watch:test
+```
```

Comment:
What’s the difference between `watch:ts` and `watch:test`? One just typechecks the TypeScript and the other one also runs the tests?

Reply:
`watch:ts` compile the ts files on every file change
`watch:test` runs the test on every file change